CSharp/Project/MVVMtpl/MVVMtpl.Shared/App.xaml.cs

```diff
@@ -133,5 +137,22 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
             // TODO: Save application state and stop any background activity
             deferral.Complete();
         }
+
+#if WINDOWS_PHONE_APP
+        /// <summary>
+        /// Invoked when back button is pressed.
+        /// </summary>
+        /// <param name="sender">The source of the suspend request.</param>
+        /// <param name="e">Details about the suspend request.</param>
+        private void HardwareButtons_BackPressed(object sender, Windows.Phone.UI.Input.BackPressedEventArgs e)
+        {
```

Comment:
Correct me if I'm wrong, or perhaps I'm missing something, but I woud say this is the default behaviour for the WP apps.